packages/builder/src/components/application/AboutProject.tsx

```diff
@@ -38,7 +38,7 @@ export function AboutProject(props: {
     : undefined;
 
   const { data: ensName } = useEnsName({
-    address: recipient ?? "",
+    address: (recipient ?? "") as `0x${string}`,
```

Comment:
This doesn't look right, since if recipient is null or undefined, then we're casting an empty string to `0x${string}`.

Reply:
True, however useEnsName is fine with both valid and invalid inputs, so not a dealbreaker here. Agree the types can be better here, but that comes with wagmi v1 and the universal proliferation of Hex or Address as a type.